src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const bigDiscount = 50;
+  const smallDiscount = 20;
+  const rentWithoutDiscount = days * 40;
+
+  if (days >= 7) {
+    return rentWithoutDiscount - bigDiscount;
+  }
+
+  if (days >= 3 && days < 7) {
+    return rentWithoutDiscount - smallDiscount;
+  }
```

Comment:
```suggestion
  if (days >= 3) {
    return rentWithoutDiscount - smallDiscount;
  }
```
Якщо попередній if не спрацював це вже говорить про те що число днів менше ніж 7